packages/docs/.storybook/test-runner.ts

```diff
@@ -0,0 +1,78 @@
+import { getStoryContext, type TestRunnerConfig } from '@storybook/test-runner';
+import { AxeBuilder } from '@axe-core/playwright';
+import { createHtmlReport } from 'axe-html-reporter';
+import assert from 'assert';
+import pc from 'picocolors';
+
+const SKIP_TESTS = 'skip-a11y';
```

Comment:
In general, it would be good to have proper doc in the docs package how to use the tags. Additional hints how to work on the bugs e. g. locally (copy path, open the A11y tab etc.).